### DISCUSSION TITLE
Error al exportar

### DISCUSSION THREAD

**Discussion by elizabethejs** [uxbox/uxbox#2735] [Q&A]
Obtengo este error en el log del exporter 

```
SEV [app.handlers.export-shapes] hint="unexpected error on export multiple"
cause data: {:type :internal, :code :timeout, :hint "ResourceRequest timed out"}
Error: ResourceRequest timed out
at new cljs.core.ExceptionInfo (/opt/app/app.js:25522:14)
at Function.cljs.core.ex_info.cljs$core$IFn$_invoke$arity$3 (/opt/app/app.js:25557:10)
at Function.app.common.exceptions.error.cljs$core$IFn$_invoke$arity$variadic (/opt/app/app.js:30307:28)
at app.browser.translate_browser_errors (/opt/app/app.js:53834:39)
at /opt/app/app.js:53027:109
SEV [app.handlers] hint="unexpected internal error"
cause data: {:type :internal, :code :timeout, :hint "ResourceRequest timed out"}
Error: ResourceRequest timed out
at new cljs.core.ExceptionInfo (/opt/app/app.js:25522:14)
at Function.cljs.core.ex_info.cljs$core$IFn$_invoke$arity$3 (/opt/app/app.js:25557:10)
at Function.app.common.exceptions.error.cljs$core$IFn$_invoke$arity$variadic (/opt/app/app.js:30307:28)
at app.browser.translate_browser_errors (/opt/app/app.js:53834:39)
at /opt/app/app.js:53027:109
```

La respuesta de la petición es
{"~:type":"~:server-error","~:code":"~:internal","~:hint":"ResourceRequest timed out","~:data":{"~:type":"~:internal","~:code":"~:timeout","~:hint":"ResourceRequest timed out"}}

**Comment by elizabethejs** [comment]
Alguien me podría ayudar con esto?

**Reply by myfunnyandy** [reply]
Hola @elizabethejs, veo que el error es en una instancia propia. ¿Podrías probar a cargar el fichero .penpot en el saas (design.penpot.app) e intentar exportarlo desde ahí? Si en ese caso no diera error, lo más probable es que sea un problema de la configuración de la instancia.

**Comment by elizabethejs** [comment]
He hecho lo que me dices y se exporta bien desde saas. alguna idea para solucionarlo?

**Comment by superalex** [comment]
Hola @elizabethejs puedes utilizar el buscador de issues y mirar algunas de las relacionadas con el exporter, casi siempre son relacionadas con temas de configuración. Por ejemplo esta: https://github.com/penpot/penpot/issues/2470

**Reply by elizabethejs** [reply]
He revisado el foro, he comprobado configuración y obtengo el mismo resultado, lo tengo desplegado en docker. Al realizar la llamada al servicio /api/exporter recibe un timeout. No se si es necesario instalar algún plugin o algo.

**Comment by elizabethejs** [comment]
Acabo de darme cuenta que aunque existan multiples elementos a la hora de exportar solo detecta uno, os ha pasado esto?
